test/e2e_node/e2e_service.go

```diff
@@ -208,6 +202,45 @@ func (es *e2eService) startServer(cmd *healthCheckCommand) error {
 	return fmt.Errorf("Timeout waiting for service %s", cmd)
 }
 
+func (es *e2eService) stopService(name string) error {
+	// Since node services are started with sudo, we must get the PID of the child process.
+	out, err := exec.Command("pidof", name).Output()
```

Comment:
I had success using `exec.Cmd.SysProcAttr.Pdeathsig`, but for some reason sometimes when the tests end I get errors like:
```
I0603 19:22:24.290494   31696 e2e_service.go:239] Killing process 5464 (kubelet) with -TERM
E0603 19:22:24.448069   31696 e2e_service.go:83] Failed to stop kubelet: error stopping kubelet: wait: no child processes
I0603 19:22:25.448432   31696 e2e_service.go:239] Killing process 5115 (kube-apiserver) with -TERM
E0603 19:22:25.608966   31696 e2e_service.go:92] Failed to stop kube-apiserver: error stopping kube-apiserver: wait: no child processes
I0603 19:22:26.609198   31696 e2e_service.go:239] Killing process 4819 (etcd) with -TERM
E0603 19:22:26.972003   31696 e2e_service.go:95] Failed to stop etcd: error stopping etcd: wait: no child processes
```
These seem to be non-deterministic (sometimes it happens, sometimes not), and I can't figure out what the cause is. Either way, it doesn't leave the services running.